code/datums/world_topic.dm

```diff
@@ -204,6 +204,7 @@
 	.["revision_date"] = GLOB.revdata.date
 	.["hub"] = GLOB.hub_visibility
 	.["identifier"] = CONFIG_GET(string/serversqlname)
+	.["public_address"] = CONFIG_GET(string/public_address) || CONFIG_GET(string/server) || "[world.internet_address]:[world.port]"
```

Comment:
this should either only trigger if the public address config is set, or ortherwise be a config that enables the behaviour since downstreams might not expect the topic to change